source/blog/en/2018-04-27-release-0.11.0.html.md

```diff
@@ -0,0 +1,23 @@
+---
+title: New version 0.11.0
+date: 2018-04-27
+---
+
+
+## New version 0.11.0 released today, with plenty of new features
+
+We have plenty of features to announce. For a complete list please visit the [changelog](https://github.com/decidim/decidim/blob/0.11-stable/CHANGELOG.md#0110pre). Here is a list of some of the most relevant new features:
+
+*   **Blogs!**: This first one you might already noticed, this post is an example :)
```

Comment:
This isn't on Decidim itself, and it can lead to confusion. Could you change it please?